app/assets/stylesheets/title.scss

```diff
@@ -0,0 +1,22 @@
+$elegant: #f0efee;
+
+h1 {
+  font-family: "Avant Garde", Avantgarde, "Century Gothic", CenturyGothic, "AppleGothic", sans-serif;
+  font-size: 30px;
+  text-align: center;
+  text-transform: uppercase;
+  text-rendering: optimizeLegibility;
+
+  &.elegantshadow {
+    color: #131313;
```

Comment:
Color literals like `#131313` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, color, letter-spacing, text-shadow